docs/process/roadmap.md

```diff
@@ -1,63 +1,70 @@
 # Roadmap
 
-The following are our marketing releases as planned to date. They contain the broad features we're considering. The further away a release is, the less defined it is. Releases are subject to change at any time.
+The following are our marketing releases, as planned to date (and we plan to revisit 6 months from now). They contain the broad features we're considering. The further away a release is, the less defined it is. Releases are subject to change at any time. This is not written in stone!!!
 
-## 1.1
+## 1.2 - Relationship between branches - lite
 
-- Relationship to _X_
-  - Show how the current branch relates to the target branch
-  - The target is the default branch or the upstream's default branch
-  - _Probably_ let the user switch the target branch
+- [Relationship between branches lite](https://github.com/desktop/desktop/issues/3956)
+  - Compare my work to any branch
+  - Merge in any other branch to my branch
+  
+- [Upgrade nudge from classic to the new app](https://github.com/desktop/desktop/issues/2146)
 
-- Conflict resolution lite
-  - Differentiate conflicted files from other changed files
-  - Let me choose my mergetool
-  - Let me open my mergetool
+## 1.3 - Detecting marge conflicts - 2.0
+  
+- [Relationship between branches 2.0](https://github.com/desktop/desktop/issues/2639)
+  - Notifications around diverging from master branch
+  - Detect merge conflicts
+  
+## 1.4 - New branch, maybe?
 
-- Pull request list
-  - _Probably_ a tab in the Branches foldout
-  - Group my pull requests vs. others
+- [Move unpublished commits on master to new branch](https://github.com/desktop/desktop/issues/1021)
+  - Don't let me commit to branches I shouldn't commit to
+  - Guide me to creating my own branches
+** depending on things, this might get combined with 1.5 
+  
+## 1.5 - Better onboarding
 
-- Pull request CI status
-  - Show PRs CI status in the above pull request list
+- Better onboarding for new users
 
-- Auto-config upstreams on fork
-  - Create an `upstream` remote for forks
-  - Fetch periodically
+## 1.6 - Conflict resolution lite
 
-## 1.2
+- [Conflict resolution lite](https://github.com/desktop/desktop/issues/2640)
+  - Differentiate conflicted files from other changed files
+  - Let me choose my mergetool
+  - Let me open my mergetool
+  
+## 1.7 - Conflict resolution - 2.0
 
-- Full conflict resolution
-  - Lots TBD
-  - Pick ours/theirs/both
+- [Conflict resolution 2.0](https://github.com/desktop/desktop/issues/2640)
   - Edit inline
   - Abort merge
   - How did I get here?
+  - Don't let me commit a file with conflict markers
+  
+## 1.8 - Messaging + Tracking
+  
+- End of Progress & in-app messaging
 
-- Fork if needed
-  - Offer to fork on clone, or push, or ?
-
-- Protected branches and default branch protection
-  - Don't let me commit to branches I shouldn't commit to
-  - Guide me to creating my own branches
+- App interaction tracking
```

Comment:
> End of progress & in-app messaging
>
> App interaction tracking

I.. totally forgot what these were. Can any shed some clarity for me? 😅

Reply:
@donokuda attempting to remember our earlier discussions:

 - "End of Progress" was calling out some of the flows where we don't make it clear that they're done - we just wrap it up and return the user to the main view. The classic example of this is login - it's not clear that your login succeeded, the flow just finishes.

 - I'm a bit hazy on "in-app messaging" and can't remember the concrete things we talked about

 - "App interaction tracking" - as we roll out features, it'd be great to understand just how much they're being used in the wild. We've been haphazard about this in the past, so this was a chance for us to revisit this.

Reply:
Crystal clear now 💎thanks, @shiftkey!